src/test/java/com/fasterxml/jackson/core/read/ArrayParsingTest.java

```diff
@@ -106,6 +107,20 @@ public void testNotMissingValueByEnablingFeature() throws Exception
         _testNotMissingValueByEnablingFeature(false);
     }
 
+    public void testDeepNesting() throws Exception
```

Comment:
Works for now: in future we probably want to create `...jacksonc.core.limits` (or `constraints`) package or something for (read) constraints